kms.tf

```diff
@@ -6,7 +6,7 @@ locals {
 }
 
 data "aws_iam_policy_document" "kms_key_global" {
-  count = var.kms.enabled && var.kms.key_id == null ? 1 : 0
+  count = local.create_kms_key
```

Comment:
@miguelhar Was this not using the local variable for any particular reason? Want to make sure I'm not overlooking some circumstance where terraform will evaluate this as unable to know until apply or something.

Reply:
Didn't initially run into it when testing, but trying to integrate tests in another branch and actually hitting it. The localness of the variable doesn't make a difference here, but I may need to change this to a list of strings or change how my test is made and provision the custom kms key to test with separately (which kind of sucks).